### PR TITLE
feat(on-host integrations): MExN integration

### PR DESCRIPTION
**Pull request by homelessbirds**

### CODE REVIEW

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: MExN integration
+tags:
+    - New Relic integrations
+metaDescription: Use New Relic browser monitoring to get a dashboard with metrics from your Flutter web application.
```
Comment (zuluecho9) [inline src/content/docs/browser/browser-integrations/mexn-integration.mdx:5]: flutter

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+---
+
+
+import browserMeanMernIntegtrationDashboard from 'images/browser_screenshot-full_mean-mern-integtration-dashboard.png'
+
+MEAN and MERN stacks are open-source full stack software development kits. Both kits use a single codebase to develop web applications. New Relic has combined both stacks into a single quickstart. Our MExN integration makes use of the Browser agent, Node,js agent, and MongoDB agent to provide you with a pre-built dashboard with your most important metrics (like response time, cpu utilization, traffic and login frequencies).
```
Comment (zuluecho9) [inline src/content/docs/browser/browser-integrations/mexn-integration.mdx:11]: have we confirmed that it's definitely a 'mongodb agent'? I've never heard of another agent besides the main ones, so I'd like to confirm from someone pretty high up that it should be called an agent and not just an integration or something else.

Comment (zuluecho9) [reply src/content/docs/browser/browser-integrations/mexn-integration.mdx:11]: CPU capitalization

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+### Step 4: Install the browser agent [#browser-install] 
+
+To install our browser agent: 
+
+1. From [one.newrelic.com](https://one.newrelic.com), go to the [**Add data** page](https://one.newrelic.com/marketplace). 
+2. Click **Browser and mobile**, then click **Browser monitoring**.
+3. Select the **Select copy/paste JavaScript code** option (for more about this code, see [our browser install docs](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent)). 
+4. Complete the rest of the steps and click **Enable.**
+5. Go the `index.html ` file of your React application
```
Comment (zuluecho9) [inline src/content/docs/browser/browser-integrations/mexn-integration.mdx:51]:
```suggestion
5. Go the `index.html ` file of your React application.
```

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+To get your MExN dashboard: 
+
+1. From [one.newrelic.com](https://one.newrelic.com), go to the [**Add data** page](https://one.newrelic.com/marketplace). 
+2. Click on **Dashboards**.
+3. In the search bar, type `MExN`. 
+4. The MExN dashboard should appear. Click on it to install it.
+
+Your MExN dashboard is considered a custom dashboard and can be found in the **Dashboards** UI. For docs on using and editing dashboards, see [our dashboard docs](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards).
+
+This integration reports data in the form of our [browser monitoring events](/docs/data-apis/understand-data/event-data/events-reported-browser-monitoring). If you installed the infrastructure agent, you'll also receive [infrastructure data](/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data/#infrastructure-events).
```
Comment (zuluecho9) [inline src/content/docs/browser/browser-integrations/mexn-integration.mdx:68]: if they're also using the MongoDB integration, I assume that means they'd also be getting events reported by that integration, which you can find in that doc. From what I'm seeing in https://docswebsitedevelop-sdmexnintegration.gatsbyjs.io/docs/infrastructure/host-integrations/host-integrations-list/mongodb/mongodb-monitoring-integration-new/#find-and-use, it seems like maybe it's `Metric` data. maybe just link to that for more info. that doc is confusing to me way it's written.

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+3. In the search bar, type `MExN`. 
+4. The MExN dashboard should appear. Click on it to install it.
+
+Your MExN dashboard is considered a custom dashboard and can be found in the **Dashboards** UI. For docs on using and editing dashboards, see [our dashboard docs](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards).
+
+This integration reports data in the form of our [browser monitoring events](/docs/data-apis/understand-data/event-data/events-reported-browser-monitoring). If you installed the infrastructure agent, you'll also receive [infrastructure data](/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data/#infrastructure-events). 
+
+Here's an example NRQL query checking the average window load time for the various browsers used by users on desktop computers: 
+
+```sql
```
Comment (zuluecho9) [inline src/content/docs/browser/browser-integrations/mexn-integration.mdx:72]: see Rob's recent advice on structuring NRQL queries. apparently we're now putting each clause on its own line. 
SELECT average(windowLoad)
FROM PageViewTiming
etc.

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx (hunk 1 above)
Comment (homelessbirds) [reply src/content/docs/browser/browser-integrations/mexn-integration.mdx:5]: done!

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx (hunk 2 above)
Comment (homelessbirds) [reply src/content/docs/browser/browser-integrations/mexn-integration.mdx:11]: done!

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx [hunk 6]
```diff
@@ -0,0 +1,93 @@
+---
+title: MExN integration
+tags:
+    - New Relic integrations
```
Comment (pkosanam) [inline src/content/docs/browser/browser-integrations/mexn-integration.mdx:4]: Can add the below two tags too
```
    - MEAN stack integration
    - MERN stack integration
```

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+### Step 4: Install the browser agent [#browser-install] 
+
+To install our browser agent: 
+
+1. From [one.newrelic.com](https://one.newrelic.com), go to the [**Add data** page](https://one.newrelic.com/marketplace). 
+2. Click **Browser and mobile**, then click **Browser monitoring**.
+3. Select the **Select copy/paste JavaScript code** option (for more about this code, see [our browser install docs](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent)). 
+4. Complete the rest of the steps and click **Enable.**
+5. Go the `index.html ` file of your React application.
```
Comment (pkosanam) [inline src/content/docs/browser/browser-integrations/mexn-integration.mdx:51]: 5. Go the `index.html ` file of your React or Angular application.

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+6.  Paste our browser code snippet script tag right before the body tag ends (`</body>`).
+8. Restart your application. 
+
+Wait five minutes and then proceed to finding your data in New Relic.
```
Comment (pkosanam) [inline src/content/docs/browser/browser-integrations/mexn-integration.mdx:55]: In couple of minutes, your application will send metrics to one.newrelic.com

**Review by pkosanam — CHANGES_REQUESTED** [review]
Also, request you to kindly download the image from our quickstart PR
image: https://github.com/newrelic/newrelic-quickstarts/blob/6960d81c673fde9c319936588d9d5e9a6e696c53/dashboards/mern/mern01.png

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx (hunk 7 above)
Comment (homelessbirds) [reply src/content/docs/browser/browser-integrations/mexn-integration.mdx:51]: done!

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx (hunk 6 above)
Comment (homelessbirds) [reply src/content/docs/browser/browser-integrations/mexn-integration.mdx:4]: done!

**Review by pkosanam — DISMISSED** [review]
looks good..
Request you to kindly update the dashboard image with the below image
image: https://github.com/newrelic/newrelic-quickstarts/blob/6960d81c673fde9c319936588d9d5e9a6e696c53/dashboards/mern/mern01.png

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+
+For information about data reported, see the docs for each of the tools we had you install:
+
+* [Browser monitoring events](/docs/data-apis/understand-data/event-data/events-reported-browser-monitoring). 
+*  [Infrastructure data](/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data/#infrastructure-events). 
+* [MongoDB data](docs/infrastructure/host-integrations/host-integrations-list/mongodb/mongodb-monitoring-integration-new/#find-and-use/).
+* [Node.js data](/docs/apm/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs/).
+
+
+Here's an example NRQL query checking the average window load time for the various browsers used by users on desktop computers:
```
Comment (nbaenam) [inline src/content/docs/browser/browser-integrations/mexn-integration.mdx:78]: This also sounds weird to me: "Here's an example NRQL query"
maybe these options?
Here's a NRQL query example checking...
Here's an example of a NRQL query

**Review by nbaenam — DISMISSED** [review]
I left some comments, feel free to add them or not. I approve it!

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+    - New Relic integrations
+    - MEAN stack integration
+    - MERN stack integration
+metaDescription: Use New Relic browser monitoring to get a dashboard with metrics from your MExN stack.
+---
+
+
+import infrastructureMeanMernIntegrationDashboard from 'images/infrastructure_screenshot-full_mean-mern-integration-dashboard.png'
+
+MEAN and MERN stacks are open-source full stack software development kits. Both kits use a single codebase to develop web applications. New Relic has combined both stacks into a single quickstart. Our MExN integration makes use of the Browser agent, Node,js agent, and MongoDB integration to provide you with a pre-built dashboard with your most important metrics (like response time, CPU utilization, traffic and login frequencies).
```
[the remark this answers is not in the document]
Comment (homelessbirds) [reply src/content/docs/browser/browser-integrations/mexn-integration.mdx:13]: Good catch! Thank you!

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx [hunk 11]
```diff
@@ -0,0 +1,95 @@
...
+
+Installing the infrastructure agent is optional, but doing this will get you important metrics about the host where your app is running. 
+
+To do this, follow [the infrastructure agent install steps](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent).
+
+### Step 2: Install MongoDB quickstart
+
+1. Check out our MongoDB agent requirements in our [documentation](/docs/infrastructure/host-integrations/host-integrations-list/mongodb-monitoring-integration/)  before installing the agent.
+2. Open the [MongoDB quickstart installation](https://newrelic.com/instant-observability/mongodb). 
+3. Click  **install now** at the bottom of the page to start the MongoDB agent installation.
```
[the remark this answers is not in the document]
Comment (homelessbirds) [reply src/content/docs/browser/browser-integrations/mexn-integration.mdx:37]: Thank you!

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+
+### Step 3: Install Node.js agent
+
+1. Check out our Node.js agent requirements in our [documentation](/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent/).
+2. Open the [Node.js quickstart installation](https://newrelic.com/instant-observability/node-js). 
+3. Click  **install now** at the bottom of the page to start the Node.js agent installation.
```
[the remark this answers is not in the document]
Comment (homelessbirds) [reply src/content/docs/browser/browser-integrations/mexn-integration.mdx:43]: Thank you!

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx [hunk 13]
```diff
@@ -0,0 +1,95 @@
...
+
+### Step 4: Install the browser agent [#browser-install] 
+
+To install our browser agent:
```
[the remark this answers is not in the document]
Comment (homelessbirds) [reply src/content/docs/browser/browser-integrations/mexn-integration.mdx:47]: 👍

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx [hunk 14]
```diff
@@ -0,0 +1,95 @@
...
+
+### Step 4: Install the browser agent [#browser-install] 
+
+To install our browser agent: 
+
+1. From [one.newrelic.com](https://one.newrelic.com), go to the [**Add data** page](https://one.newrelic.com/marketplace). 
+2. Click **Browser and mobile**, then click **Browser monitoring**.
+3. Select the **Select copy/paste JavaScript code** option (for more about this code, see [our browser install docs](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent)). 
+4. Complete the rest of the steps and click **Enable.**
+5. Go the `index.html ` file of your React application or Angular application.
```
[the remark this answers is not in the document]
Comment (homelessbirds) [reply src/content/docs/browser/browser-integrations/mexn-integration.mdx:53]: 👍

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx [hunk 15]
```diff
@@ -0,0 +1,95 @@
...
+To get your MExN dashboard: 
+
+1. From [one.newrelic.com](https://one.newrelic.com), go to the [**Add data** page](https://one.newrelic.com/marketplace). 
+2. Click on **Dashboards**.
+3. In the search bar, type `MExN`. 
+4. The MExN dashboard should appear. Click on it to install it.
+
+Your MExN dashboard is considered a custom dashboard and can be found in the **Dashboards** UI. For docs on using and editing dashboards, see [our dashboard docs](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards).
+
+For information about data reported, see the docs for each of the tools we had you install:
```
[the remark this answers is not in the document]
Comment (homelessbirds) [reply src/content/docs/browser/browser-integrations/mexn-integration.mdx:70]: thanks!

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx (hunk 9 above)
Comment (homelessbirds) [reply src/content/docs/browser/browser-integrations/mexn-integration.mdx:78]: 👍

**Review by nbaenam — APPROVED** [review]
It's perfect!!!

**Pull request merged**